### DISCUSSION TITLE
About the SOPS's installation process on a Windows 10 machine

### DISCUSSION THREAD

**Discussion by ozanarifyavuz** [getsops/sops#1251] [Q&A]
Hello Dear SOPS community, 

First off, I want to thank all of you for this beautiful project, and greetings from Vienna! :)

I have started to work as student software engineer in a company, and I am a part of infrastructure team. My first task is to encrypt the keys in the files using SOPS, but I couldn't manage to download and install SOPS on my computer. I have tried to install **sops-v3.7.3.exe** file but it didn't work. In addition, my computer gave me error when I tried to download and install the other installation-related files, and they didn't work either.

Can you please help me with this?

Yours sincerely,
Ozan Arif Yavuz

[a comment by an automated account is not shown]

**Comment by petersimkovic** [comment]
Hi Ozan,
I had the same issue and after some trying I found out that scoop is pretty amazing for this:
`iwr -useb get.scoop.sh | iex`
`scoop update`
`scoop install sops`

I hope that you don't need that anymore, but hopefully it might help other help seekers.

**Comment by Pavel-Nova** [comment]
You can configure SOPS tool on Windows manually: 
1. Download latest [sops-v3.9.1.exe](https://github.com/getsops/sops/releases/download/v3.9.1/sops-v3.9.1.exe)
2. Copy the tool to the _C:\Program Files (x86)\Sops\v3.9.1_ folder and rename to _sops.exe_
3. Open "Environment Variables" window and add _C:\Program Files (x86)\Sops\v3.9.1_ to the PATH variable
4. Open CMD and try: `sops -v`
